Cargo.toml

```diff
@@ -11,13 +11,14 @@ members = [
     "vortex-error",
     "vortex-expr",
     "vortex-flatbuffers",
+    "vortex-scalar",
     "vortex-serde",
     "vortex-sampling-compressor",
 ]
 resolver = "2"
 
 [workspace.package]
-version = "0.1.0"
+version = "0.2.0"
```

Comment:
Should this bump be here?

Reply:
I think we can.

When you do `cargo publish`, it uses the cargo metadata from crates.io to find things like features, rather than examining locally. But I just checked and release-plz does a topological sort and auto-publishes everything in the right order, so I think by the time each package is publishing, its dependencies are already gonna be on crates.io